microservices-connector/config/manifests/tgi_gaudi.yaml

```diff
@@ -107,17 +107,20 @@ spec:
             periodSeconds: 5
             tcpSocket:
               port: http
+            timeoutSeconds: 1
           readinessProbe:
             initialDelaySeconds: 5
             periodSeconds: 5
             tcpSocket:
               port: http
+            timeoutSeconds: 1
           startupProbe:
-            failureThreshold: 120
+            failureThreshold: 180
```

Comment:
Failure thresholds can (and should) be much smaller on accelerators (gaudi/nv), only CPU version needs extended thresholds, as it can _much_ longer to start.

Reply:
done